AerofoilMac/AerofoilMac/AerofoilMac.entitlements

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.app-sandbox</key>
+	<true/>
+	<key>com.apple.security.cs.disable-library-validation</key>
```

Comment:
Actually one more thing: What are these entitlements needed for?  SDL?

And I don't think there are any cases where the file access permission should be needed any more, all file access should be constrained to the app's user directory, and the app's own data, so I don't think there's anything that uses a file selection dialog any more, is there?

Reply:
com.apple.security.cs.disable-library-validation is needed for SDL. For notarization you need to enable hardened mode. And this 'exception' is needed for SDL to work. Does not change anything for the notarization. Guess its just there to prevent apps from having privileges they don't need.

com.apple.security.files.user-selected.read-only was there by default, not something I put in specifically.

com.apple.security.app-sandbox is also default true, and means the app is sandboxed.